versioned_docs/version-0.1.0/notifications/notifications-mail.md

```diff
@@ -0,0 +1,24 @@
+---
+id: notifications-mail
+slug: /notifications-mail
+sidebar_position: 1
+---
+# Email configuration
+
+
+MoonGuard uses the Mail configuration that the user's app has in their project,
+and it is not possible to send notifications via email without it.
+
+Once you have config your email, add the following variables of the email in the
+project `.env` file:
+
+```php
+MAIL_MAILER=smtp
+MAIL_HOST=smtp.mailtrap.io
+MAIL_PORT=2525
+MAIL_USERNAME=xxxxxxxxxxx
+MAIL_PASSWORD=xxxxxxxxxxx
+MAIL_ENCRYPTION=tls
+MAIL_FROM_ADDRESS="demo@example.com"
+MAIL_FROM_NAME="${APP_NAME}"
+```
```

Comment:
# Mail configuration

MoonGuard uses the mail configuration from your Laravel app to send any email notification. 

```php
MAIL_MAILER=smtp
MAIL_HOST=mailpit
MAIL_PORT=1025
MAIL_USERNAME=null
MAIL_PASSWORD=null
MAIL_ENCRYPTION=null
MAIL_FROM_ADDRESS="hello@example.com"
MAIL_FROM_NAME="${APP_NAME}"
```

After properly setting up the parameters of your mail provider, MoonGuard is able to send email notifications.